test/functional/page_objects/settings_page.js

```diff
@@ -62,53 +62,56 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
       await PageObjects.common.navigateToApp('settings');
     }
 
-    getIndexPatternField() {
-      return testSubjects.find('createIndexPatternNameInput');
+    async getIndexPatternField() {
+      return await testSubjects.find('createIndexPatternNameInput');
     }
 
-    getTimeFieldNameField() {
-      return testSubjects.find('createIndexPatternTimeFieldSelect');
+    async clickTimeFieldNameField() {
+      return await testSubjects.click('createIndexPatternTimeFieldSelect');
+    }
+
+    async getTimeFieldNameField() {
+      return await testSubjects.find('createIndexPatternTimeFieldSelect');
     }
 
     async selectTimeFieldOption(selection) {
       // open dropdown
-      (await this.getTimeFieldNameField()).click();
+      await this.clickTimeFieldNameField();
       // close dropdown, keep focus
-      (await this.getTimeFieldNameField()).click();
+      await this.clickTimeFieldNameField();
       await PageObjects.header.waitUntilLoadingHasFinished();
-      await retry.try(async () => {
+      return await retry.try(async () => {
```

Comment:
So I could have sworn you gave me feedback once that said I should add a return for an async function, e.g. like I'm doing above:
```
    async clickTimeFieldNameField() {
       return await testSubjects.click('createIndexPatternTimeFieldSelect');
    }
```

The return isn't necessary then either, right?  Perhaps I misunderstood your original comment from awhile back.